src/popup.js

```diff
@@ -0,0 +1,206 @@
+    footer.classList.remove('d-none');
+    listItemsContainer.classList.remove('recipe-section');
+    document.body.style.overflowY = 'scroll';
+  });
+
+  popupBox.classList.add('ml-auto', 'mr-auto', 'pb-5');
+  popupBox.append(close, title, photo);
+
+  parentContainer.classList.add('parent-container');
+  title.classList.add('popup-title');
+  photo.classList.add('popup-img');
+
+  popupBox.classList.add('ml-auto', 'mr-auto', 'pb-5');
+  popupBox.append(title, photo);
+
+  const commentCont = document.createElement('div');
+  const dataList = document.createElement('ul');
+  const commentlist = document.createElement('ul');
+  const commentsTitle = document.createElement('h3');
+  const commentwrapper = document.createElement('li');
+
+  dataList.classList.add('popup-info');
+
+  commentlist.append(commentsTitle);
+
+  commentCont.classList.add('text-justify');
+  commentCont.append(dataList, commentlist);
+
+  popupCont.classList.add('p-3', 'm-2', 'd-flex', 'flex-column');
+  popupCont.append(popupBox, commentCont);
+
+  const socialCont = document.createElement('div');
+
+  const formCont = document.createElement('form');
+
+  formCont.setAttribute('id', 'form');
+
+  const formInside = document.createElement('div');
+  const buttonCont = document.createElement('div');
+
+  const button = document.createElement('button');
+  button.innerHTML += 'Add Comment';
+  button.classList.add('nav-link', 'comment-btn', 'align-middle', 'p-1');
+
+  buttonCont.classList.add('column', 'pt-2', 'ml-2', 'mr-2');
+  buttonCont.appendChild(button);
+
+  const nameInput = document.createElement('input');
+  nameInput.required = true;
+  nameInput.setAttribute('id', 'nameInput');
+  nameInput.setAttribute('placeholder', 'Name');
+  nameInput.classList.add('form-control', 'm-2');
+  const commentArea = document.createElement('textarea');
+  commentArea.required = true;
+  commentArea.setAttribute('id', 'comment');
+  commentArea.setAttribute('placeholder', 'Leave a comment');
+  commentArea.classList.add('form-control', 'm-2', 'form-c');
+
+  ApiData.getMeal(foodId).then((data) => {
+    const info = data.meals[0];
+    const foodName = info.strMeal;
+    const foodImg = info.strMealThumb;
+    const foodTag = info.strTags;
+    const numero = info.idMeal;
+    const instructions = info.strInstructions;
+
+    title.innerHTML += foodName;
+    title.classList.add('text-center');
+
+    photo.src = foodImg;
+
+    const comm = document.createElement('li');
+
+    comm.classList.add('popup-tags');
+
+    if (foodTag !== null) {
+      const splitTags = foodTag.split(',');
+      splitTags.forEach((tag) => {
+        const tagCont = document.createElement('h4');
+        tagCont.innerHTML += `#${tag} `;
+        comm.appendChild(tagCont);
+      });
+    } else {
+      comm.innerHTML += 'Delicious Food';
+    }
+
+    dataList.appendChild(comm);
+
+    const filler = () => {
+      commentsApi.fetchRecipes(numero, appID).then((data) => {
```

Comment:
- [ ] This request fails when the popup is displayed. You can fix this
![Screenshot from 2021-10-15 00-57-44](https://user-images.githubusercontent.com/65476715/137411824-205491eb-5eac-4135-b1be-18ea1026e9e2.png)

Reply:
Hi @ramiaquista, Sorry for the late response. I hope you have been able to make progress.